app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationInfo.jsx

```diff
@@ -152,7 +300,13 @@ export const IntegrationInfo = (props) => {
       {!integrationId ? (
         integrationContent()
       ) : (
-        <h1>Configuration Page with unique id = {integrationId}</h1>
+        <>
+          <h1>Configuration Page with unique id = {integrationId}</h1>
+          <div className={cx('buttons-setting')}>
+            <Button onClick={editAuthorizationClickHandler}>Edit</Button>
+            <Button onClick={onDeleteProjectIntegration}>Delete</Button>
```

Comment:
Looks like we need to use here formatteMessage function. Because now it does not translates into ru be ua.

Reply:
it is just temporary solution